preview-client/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "archive-unzip-client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "kmedia-mdb": "https://github.com/Bnei-Baruch/kmedia-mdb.git",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-scripts": "4.0.1",
+    "@material-ui/core": "^4.11.2",
+    "@material-ui/icons": "^4.11.2",
+    "jquery": "^3.5.1"
```

Comment:
Really ?! 
It's 2021. Are we still using JQuery ?

Reply:
removed